qcodes/instrument/parameter.py

```diff
@@ -339,6 +339,16 @@ def __init__(self, name: str,
         # to perform a validation on get
         self._validate_on_get = False
 
+    @property
```

Comment:
For readablilty should we put these next to full_name ?

Reply:
Yes. Done as suggested